qwesty.ino

```diff
@@ -210,7 +210,7 @@ void loop() {
     setOctave(keyCode);
     updatePendingHeldNotesCount();
     lcdUpdateRootAndOctave();
-  } else if (isRoot(keyCode)) {
+  } else if (isRoot(keyCode) || isIncrementalTranspose(keyCode)) {
```

Comment:
let's keep the incremental vs keypad transpose code blocks separate conditions. See later comment.